docs/tutorial.md

```diff
@@ -469,29 +459,36 @@ spec:
         name: skaffold-image-leeroy-web
 ```
 
-The `PipelineRun` will create the `TaskRuns` corresponding to each `Task` and
-collect the results.
+The `PipelineRun` automatically defines a corresponding `TaskRun` for each `Task` you've defined
+in your `Pipeline`. It also  and collect the results of running each `Task`. In our example, the
+`TaskRun` order is as follows:
+
+1. `tutorial-pipeline-run-1-build-skaffold-web` runs `build-skaffold-web`,
+   since it has no [`from` or `runAfter` clauses](pipelines.md#ordering).
+1. `tutorial-pipeline-run-1-deploy-web`  runs `deploy-web`, because
+   its [input](tasks.md#inputs) `web-image` comes [`from`](pipelines.md#from)
+   `build-skaffold-web`. Thus, `build-skaffold-web` must run before `deploy-web`.
 
-To apply the yaml files use the following command, you will need to apply the
-`deploy-task` if you want to run the Pipeline.
+Apply the above YAML files as follows:
 
 ```bash
 kubectl apply -f <name-of-file.yaml>
 ```
+**Note:** Also apply the `deploy-task` or the `PipelineRun` will not execute.
```

Comment:
Should be mentioned before the apply command. My suggestion above covers this but could be more explicit to mention the name of the Task, Pipeline.